source/common/runtime/runtime_features.cc

```diff
@@ -135,6 +135,9 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_enable_universal_header_validator)
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_quic_defer_logging_to_ack_listener);
 // TODO(#31276): flip this to true after some test time.
 FALSE_RUNTIME_GUARD(envoy_restart_features_use_fast_protobuf_hash);
+// Execution context is optional and must be enabled explicitly.
+// See https://github.com/envoyproxy/envoy/issues/32012.
+FALSE_RUNTIME_GUARD(envoy_restart_features_enable_execution_context);
```

Comment:
The flags are ordered by the alphabet order. So, this should be placed at the top of the exist restart flag.